src/components/transactions/LiGovTransaction.vue

```diff
@@ -49,35 +39,30 @@
           Proposal &#35;{{ tx.proposal_id }}
         </router-link>
       </div>
-      <div slot="fees">
-        Network Fee:&nbsp;<b>{{ convertedFees ? convertedFees.amount : 0 }}</b>
-        <span>
-          {{
-            convertedFees
-              ? viewDenom(convertedFees.denom)
-              : viewDenom(bondingDenom)
-          }}
-        </span>
-      </div>
     </template>
   </LiTransaction>
 </template>
 
 <script>
 import LiTransaction from "./LiTransaction"
-import num, { atoms, viewDenom } from "../../scripts/num.js"
+import { atoms, viewDenom, shortDecimals } from "../../scripts/num.js"
 
 export default {
   name: `li-gov-transaction`,
   components: { LiTransaction },
+  filters: {
+    toAtoms: atoms,
+    viewDenom: viewDenom,
```

Comment:
```suggestion
    viewDenom,
```

Reply:
Good spot.

I now also import `atoms` using the `…as toAtoms`.